packages/grafana-schema/src/common/common.gen.ts

```diff
@@ -722,6 +722,16 @@ export enum TableCellBackgroundDisplayMode {
   Gradient = 'gradient',
 }
 
+/**
+ * Whenever we add text wrapping, we should add all text wrapping options at once
+ */
+export interface TableWrapTextOptions {
+  /**
+   * if true, wrap the text content of the cell
+   */
+  wrapText?: boolean;
+}
```

Comment:
Levitate thinks this is a breaking change; it's not, since all of the same interfaces are still extended by `wrapText`.